helm-charts/charts/kube-starrocks/charts/starrocks/values.yaml

```diff
@@ -127,6 +127,17 @@ starrocksFESpec:
     repository: starrocks/fe-ubuntu
     tag: ""
   imagePullPolicy: IfNotPresent
+  # Specify the entrypoint for FE.
+  # By default, operator will use '/opt/starrocks/fe_entrypoint.sh' as command, and use '$(FE_SERVICE_NAME)' as args in container spec.
+  # If entrypoint is set, the command will be ["bash", "-c"], and the args will be filename of the entrypoint script.
+  # Pod will be restarted if the entrypoint script is updated.
+  entrypoint: {}
+    # script: |
+    #   #! /bin/bash
+    #   echo "do something before start fe"
+    #   exec /opt/starrocks/fe_entrypoint.sh $FE_SERVICE_NAME
+    # # a configmap with name $cluster-fe-entrypoint-script will be created, and the script will be mounted to /etc/init-script/entrypoint.sh
+    # mountPath: /etc/init-script
```

Comment:
will this `mountPath` critical to end user, or this can be completely transparent to the end user.

Reply:
For users, it is indeed unnecessary to specify. We can hide it.

Reply:
done